src/cbc_sdk/base.py

```diff
@@ -909,8 +909,8 @@ def _refresh_if_needed(self, request_ret):
             except Exception:
                 message = request_ret.text
```

Comment:
Can we rename the variable to result instead of message

Reply:
Yes